client/lbclient/boot/replication.js

```diff
@@ -51,4 +53,8 @@ module.exports = function(client) {
   });
 
   client.sync = sync;
+
+  client.getSimpleModelId = function(modelId) {
```

Comment:
While you are at changing this, could you please use a more descriptive adjective? E.g. `getReadableModelId` or `getPronouncableId`.